src/platforms/common/configuration/options.mdx

```diff
@@ -841,18 +841,38 @@ Set this boolean to `false` to disable the scope sync from Java to NDK on Androi
 Set this boolean to `true` to automatically attach all threads to all logged events on Android.
 </ConfigKey>
 
-<ConfigKey name="enableAutoPerformanceTracking" supported={["react-native", "flutter"]}>
+<ConfigKey name="enableAutoPerformanceTracking" supported={["flutter"]}>
 
 Set this boolean to `false` to disable auto [performance monitoring](/product/performance/) tracking.
 
 </ConfigKey>
 
-<ConfigKey name="enableOutOfMemoryTracking" supported={["react-native", "javascript.capacitor", "flutter"]}>
+<ConfigKey name="enableAutoPerformanceTracing" supported={["react-native"]}>
+
+<Note>
+Available since version 8.0.0 of Sentry Apple SDK and 5.0.0 of Sentry React Native SDK. It was named `enableAutoPerformanceTracking` before the mentioned releases.
+</Note>
```

Comment:
```suggestion
<Note>
The `enableAutoPerformanceTracking` config key has been available since version 8.0.0 of Sentry's Apple SDK and version 5.0.0 of Sentry's React Native SDK. 
</Note>
```
Let me know if this changes the meaning of what you were trying to say too much.

Reply:
I've decided, that mentioning only the `cocoa` version will be enough. As rn depends on the cocoa.